packages/Slider/index.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { render } from '../../utils/tests'
+
+import { Slider } from './index'
+
+describe('<Slider>', () => {
+  it('should render correctly', () => {
+    const handleChange = jest.fn()
+    const { container } = render(<Slider max={100} min={0} onChange={handleChange} value={50} />)
+
+    expect(container)
+  })
```

Comment:
Maybe you could add more tests for the Slider component? Check if it correctly renders range if you add it, if the value changes when arrows left or right are pressed for instance...?

Reply:
Yes, I wanted to ask a front developer from another squad to help me doing some tests since I don't have a lot of experience doing it :D

Reply:
Hey @Leiksa, I can help you with your test if you want, maybe on Wednesday? What you can do by then is writing all the test scenarii you would like to experiment :) And we'll try to implement them together